app/views/samples/show.html.erb

```diff
@@ -1,30 +1,28 @@
-<div id="PipelineSampleReads_page">
+<div id="sample_view">
   <%= javascript_tag do %>
-    react_component('PipelineSampleReads', {
+    react_component('SampleView', {
       admin: <%= current_user.admin %>,
       allowedFeatures: JSON.parse('<%= raw escape_json(current_user.allowed_feature_list) %>'),
```

Comment:
Just a question have we set on camelCase vs snake_case for the JSONs? 
Or did you just tried to make this uniform.

Reply:
I tried to make it uniform. We haven't formally decided on it. Although...since these keys are passed into the components as props, and we generally want props to be camelCase, I think this is a reasonable convention. Let me know what you think/

Reply:
We cannot meet both sides. Ruby use snake_case and Javascript camelCase.
We just have to define a convention to follow in the future.
I would prefer camelCase just because it is easy to convert camelCase->snake_case in rails if we want to using [deep_transform_keys](https://apidock.com/rails/v4.0.2/Hash/deep_transform_keys) and [underscore](https://apidock.com/rails/ActiveSupport/Inflector/underscore).